ksqldb-parser/src/main/java/io/confluent/ksql/util/ParserUtil.java

```diff
@@ -256,7 +256,7 @@ public static ColumnConstraints getColumnConstraints(
     } else if (context.HEADERS() != null) {
       builder = builder.headers();
     } else if (context.HEADER() != null) {
-      builder = builder.header(Strings.trimToNull(unquote(context.STRING().getText(), "'")));
```

Comment:
Can you trim the text after the `unquote` call? The case was that a user could use `header('  k  ')`, and that key should be `k`. The `trimToNull` justs makes sure that `header('   ')` is null.

Reply:
I see. Thanks for the catch!